apps/src/applab/ai.js

```diff
@@ -56,6 +56,7 @@ function generateCodeDesignElements(modelId, modelData) {
   designMode.updateProperty(predictButton, 'id', predictButtonId);
   var predictOnClick = `onEvent("${predictButtonId}", "click", function() {
     ${inputFields.join('\n\t\t')}
+    setText("${predictionId}", '');
```

Comment:
It seems as though this relies solely on the prediction delay for the visible blanking?  Could it be too fast to notice in some cases?  If so, could we introduce a minimum time of blankness regardless of prediction speed?  (This is a common pattern, sometimes used for showing a spinner for a minimum amount of time for operations of unpredictable duration.)

Reply:
Ah, adding that kind of complexity to generated code would probably be overwhelming for the student, which I had overlooked when making the above comment.  The one thing we could do is add an arbitrary delay to the time that `getPrediction` takes, but we could ponder that later.

Reply:
Sounds good! I'll keep an eye out for this during the next classroom visit.